Short-Answer/Algorithms_Questions.md

```diff
@@ -27,10 +31,28 @@ c)  def bunnyEars(bunnies):
         return 0
 
       return 2 + bunnyEars(bunnies-1)
+      # This is 0(1) as it no matter the input size, it will only ever return one thing, as it only accepts 0 anyways
 ```
 
 ## Exercise II
 
 Suppose that you have an n-story building and plenty of eggs. Suppose also that an egg gets broken if it is thrown off floor f or higher, and doesn't get broken if dropped off a floor less than floor f. Devise a strategy to determine the value of f such that the number of dropped + broken eggs is minimized.
 
 Write out your proposed algorithm in plain English or pseudocode AND give the runtime complexity of your solution.
+
+STRATEGY:
+n = 20
+f = ?
+e = 0
+Start e at zero
+Start n at the number of floors
+Start f at the floor where eggs get broken
+Loop through the length of n floors and increase e+1 through each iteration until e = f
+
```

Comment:
We want to binary search to see where the breaking point is. The time complexity of an egg drop can be found in 
 `O( log n )`